weatherman.py

```diff
@@ -0,0 +1,194 @@
+import argparse
+import calendar
+from colorama import Fore, Style
+import csv
+import glob
+import os
+
+def convert_month(weather_month_input):
+    try:
+        month_number = int(weather_month_input)
+        if 1 <= month_number <= 12:
+            return calendar.month_abbr[month_number]
+    except ValueError:
+        raise ValueError("Invalid input")
+    
+def read_files(weather_year, weather_month):
```

Comment:
read files don't define the what kind of file we are going to read, it should be `read_weather_files`